### PR TITLE
Add missing dependencies and update

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-web3/test/ULWeb3Provider.test.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ describe('ULWeb3Provider', () => {
   describe('get accounts', () => {
     it('initialize onboarding when wallet does not exist', async () => {
       const initOnboardingSpy = sinon.spy(() => null);
-      sinon.replace(ulProvider, 'initOnboarding', initOnboardingSpy);
+      sinon.replace(ulProvider, 'initOnboarding', initOnboardingSpy as any);
```
Comment (Marik-D) [inline universal-login-web3/test/ULWeb3Provider.test.ts]: Why is this needed?

Comment (xitronix) [reply universal-login-web3/test/ULWeb3Provider.test.ts]: I resolved it. Initialization should return a Promise, not null
